samples/enumdevdir.c

```diff
@@ -16,31 +16,47 @@
 #include <cc65.h>
 
 
-void printdir (char *newdir)
+int printdir (char *newdir)
 {
-    char olddir[FILENAME_MAX];
-    char curdir[FILENAME_MAX];
+    char *olddir;
+    char *curdir;
     DIR *dir;
     struct dirent *ent;
     char *subdirs = NULL;
     unsigned dirnum = 0;
     unsigned num;
 
-    getcwd (olddir, sizeof (olddir));
+    olddir = malloc (FILENAME_MAX);
+    if (olddir != NULL) {
+
```

Comment:
When another commit is necessary anyway please remove this blank line.

Reply:
Hmm. I added this blank line since after the `if (chdir (newdir))` there is one.